### PR TITLE
@feature/86ep326h3/dnd-validation ft code enhancement

### PR DESCRIPTION
**Pull request by DropitupQwerty**
- Deletion of unused files
- Creation of Reusable Component
- Adding types

### CODE REVIEW

File: apps/simulator/src/components/blocks/CaseStudy/CaseStudyQuestions/DNDComponent/WordDropContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+type Props = {
+    placeholder: string;
+    name: string;
+    setWord: (items: DND1WordChoicesUI[]) => void;
+    onDelete: (item: DND1WordChoicesUI) => void;
+    onDropped: (item: DND1WordChoicesUI) => void;
+}
+
+export const WordDropContainer: React.FC<Props> = ({ placeholder, name, onDelete, onDropped, setWord }) => {
```
Comment (JMSevilla) [inline apps/simulator/src/components/blocks/CaseStudy/CaseStudyQuestions/DNDComponent/WordDropContainer.tsx:15]: prevent prop drilling, use ...rest

File: apps/simulator/src/core/types/ssrData.ts [hunk 2]
```diff
@@ -45,6 +45,31 @@ export type AnswerProps = {
...
 };
 
+export type DND1WordChoicesUI = {
+  id: number;
+  text: string;
+}
+
+export type DND1Object = {
+  DNDID: number;
+  DND1WordChoice1: Array<any>;
```
Comment (JMSevilla) [inline apps/simulator/src/core/types/ssrData.ts:55]: Why we are using `Array<any>` ?

File: packages/utils/types/preloaded-types/preloaded-types-responses/index.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ export type AnswerUIItem = {
   qId: number;
   answerType: string;
   answerInstruction: string;
-  rows: string;
+  rows: string | Array<any>;
```
Comment (JMSevilla) [inline packages/utils/types/preloaded-types/preloaded-types-responses/index.ts:34]: If rows can store different values of data then you can just use string | any or any.

**Review by JMSevilla — CHANGES_REQUESTED** [review]
Clarification and request changes.

File: packages/utils/types/preloaded-types/preloaded-types-responses/index.ts (hunk 3 above)
Comment (DropitupQwerty) [reply packages/utils/types/preloaded-types/preloaded-types-responses/index.ts:34]: My Bad po Sir in this area. Di po talaga dapat sya included sa push ko

**Review by JMSevilla — APPROVED** [review]

**Review by JezChristian — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

**Pull request merged**